gcp/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# syntax=docker/dockerfile:1
+FROM gcr.io/google.com/cloudsdktool/cloud-sdk:slim
+
+ENV PYTHONUNBUFFERED True
+ENV GOOGLE_APPLICATION_CREDENTIALS ./creds.json
```

Comment:
> Is this secure?

Unless an unauthorized party gets access to the container registry, no data will be compromised or leaked

> How to test security "leakage" ?

An unauthorized party with access to the container will be able to view the `creds.json` file. Some form of credentials are required to authorize the container for GCP. A leakage will occur if a user gets access to Google's Container Registry, the JSON bucket, or Cloud Run

> If creds age does container need to be re-built?

Yes, the container needs to be re-built since the `creds.json` file is used to access the GCP apps